.gitignore

```diff
@@ -33,3 +33,4 @@ dist/
 
 doc/GREADME.md
 doc/docMap.json
+package-lock.json
```

Comment:
I would add this file to the repo instead: https://github.com/canjs/can-reflect/blob/master/.npmrc

Reply:
Added!